netlify.toml

```diff
@@ -8,7 +8,7 @@ publish = "website/public"
 
 # Our Makefile builds Hugo, but it's faster if Netlify grabs the correct version on their own
 # via a simple cURL.
-environment = { HUGO_VERSION="v0.101.0" }
+environment = { HUGO_VERSION="v0.109.0" }
```

Comment:
I think you changed the hugo version for netlify here to v0.109, but locally you're still using v0.101.
You can upgrade the version you are using locally using `bingo get github.com/gohugoio/hugo@v0.109.0`

That way you'd be able to replicate the netlify error locally.

Reply:
bingo command is not working. I have installed the bingo using `go install` . Also, the reason I updated the hugo version in netlify to v0.109.0  as there was same problem in v0.101.0, so thought it might that .